### PR TITLE
[stable/kiam] Update to latest kiam image release (3.4)

### PR DESCRIPTION
**Pull request by jeffb4**
#### Is this a new chart

No

#### What this PR does / why we need it:

Updates to latest release of kiam (3.4, released August 16th)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/kiam/values.yaml [hunk 1]
```diff
@@ -137,7 +137,7 @@ server:
 
   image:
     repository: quay.io/uswitch/kiam
-    tag: v3.3
+    tag: v3.4
```
Comment (davidkarlsen) [inline stable/kiam/values.yaml:140]: can you update the readme too - so that it matches? TIA

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/kiam/values.yaml (hunk 1 above)
Comment (jeffb4) [reply stable/kiam/values.yaml:140]: Done

**Comment by ewbankkit** [comment]
@jeffb4 The kiam chart is now hosted by uSwitch.
Please open the PR in their repo and close here.
Thanks.

**Comment by jeffb4** [comment]
Closing per request

**Pull request closed without merging**